glossary.po

```diff
@@ -1072,7 +1072,7 @@ msgstr ""
 "Fait généralement référence à une fonction générateur mais peut faire "
 "référence à un *itérateur de générateur* dans certains contextes. Dans les "
 "cas où le sens voulu n'est pas clair, utiliser les termes complets lève "
-"l’ambigüité."
+"l’ambiguïté."
```

Comment:
Les deux sont valables 😉 
`ambiguïté (Réf. ortho. ambigüité)`
https://www.larousse.fr/dictionnaires/francais/ambigu%C3%AFt%C3%A9/2720

Reply:
Comme grammalecte le remonte, je l'ai corrigé. Ca évite d'avoir des faux positifs avec make verifs je pense ;)